recipes/emela/meta.yaml

```
@@ -0,0 +1,125 @@
+{% set name = "emela" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/gstagnit/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 53a867616f7ae0b03928c5a9db4b1b04a6b29794c9492a96b19dfea64b1c856d
+  folder: source
+  patches:
+    - 0001-support-range-of-cmakes.patch
+
+build:
+  skip: true  # [win]
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('emela', max_pin='x.x') }}
+  script:
+    - cmake -DCMAKE_INSTALL_PREFIX=$PREFIX -DWITH_LHAPDF=ON -S source -B build
+    - cmake -LH build
+    - cmake --build build --clean-first --parallel="${CPU_COUNT}"
+    - cmake --install build
+
+requirements:
+  build:
+    - {{ stdlib('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - cmake
+    - make
+  host:
+    - libboost-devel <1.85
```

Leave this unpinned as this should use 1.84: https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/9fb8cba137e5cb9892796f7fb9df56a63fc420c2/recipe/conda_build_config.yaml#L469
```suggestion
    - libboost-devel
```

---

Okay, I'll do so. If I can't get thing fixed upstream before `v1.85` is used I can reasonably add pins then (though still discouraged), yes?

---

Yes, you can add a pin but please us a fixed version then, e.g. 1.84. We have a migration running for 1.86, simply don't merge or close the PR as long as the package doesn't support it. Once the migration is finalized, you will need the pin but boost migration take quite long.

If you start out with a pin here, the migration PR will never be issued.

---

> If you start out with a pin here, the migration PR will never be issued.

Ah! I didn't know this. Thanks!